guides/common/modules/con_dhcp-service-providers.adoc

```diff
@@ -0,0 +1,28 @@
+[id="dhcp-serivce-proviers"]
+= DHCP service providers
+
+{Project} can manage IP leases on a DHCP server by using {SmartProxy}.
```

Comment:
I'll leave this to writers, but given a Smart Proxy is essentially a facade, would this be better?
```suggestion
{Project} can manage IP leases on a DHCP server through a {SmartProxy}.
```